ssv/spectest/generate/main.go

```diff
@@ -72,12 +75,23 @@ func clearStateComparisonFolder() {
 }
 
 func writeJsonStateComparison(name, testType string, post interface{}) {
+	postMap, ok := post.(map[string]types.Root)
+
+	if ok {
+		for subTestName, postState := range postMap {
+			writeSingleSCJson(filepath.Join(name, subTestName), testType, postState)
```

Comment:
was that done for readability? or?

Reply:
This makes multi-tests output jsons files per subtest.
It makes the world of reviewing states way easier.

It might be o a bit out of scope of this PR.
But it was too hard for me to work w.o this.